driver/src/main/java/org/neo4j/driver/internal/cluster/ClusterRoutingTable.java

```diff
@@ -35,21 +35,26 @@
     private static final int MIN_ROUTERS = 1;
 
     private final Clock clock;
-    private volatile long expirationTimeout;
+    private volatile long expirationTimestamp;
     private final AddressSet readers;
     private final AddressSet writers;
     private final AddressSet routers;
 
-    public ClusterRoutingTable( Clock clock, BoltServerAddress... routingAddresses )
+    private final String databaseName; // specifies this routing table is the routing table of database named this.
```

Comment:
maybe a clearer comment like `// specifies the database this routing table is acquired for`?